File: sdkwork-app-sdk-java/src/main/java/com/sdkwork/app/model/ImageStatisticsVO.java
package com.sdkwork.app.model;

public class ImageStatisticsVO {
    private String createdAt;
    private String updatedAt;
    private Integer totalImages;
    private Integer completedImages;
    private Integer processingImages;

    public String getCreatedAt() {
        return this.createdAt;
    }
    
    public void setCreatedAt(String createdAt) {
        this.createdAt = createdAt;
    }

    public String getUpdatedAt() {
        return this.updatedAt;
    }
    
    public void setUpdatedAt(String updatedAt) {
        this.updatedAt = updatedAt;
    }

    public Integer getTotalImages() {
        return this.totalImages;
    }
    
    public void setTotalImages(Integer totalImages) {
        this.totalImages = totalImages;
    }

    public Integer getCompletedImages() {
        return this.completedImages;
    }
    
    public void setCompletedImages(Integer completedImages) {
        this.completedImages = completedImages;
    }

    public Integer getProcessingImages() {
        return this.processingImages;
    }
    
    public void setProcessingImages(Integer processingImages) {
        this.processingImages = processingImages;
    }
}

File: sdkwork-app-sdk-typescript/src/types/image-statistics-vo.ts
/** 图片统计响应 */
export interface ImageStatisticsVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 总图片数 */
  totalImages?: number;
  /** 已完成图片数 */
  completedImages?: number;
  /** 处理中图片数 */
  processingImages?: number;
}
